### PR TITLE
Update takePicture() docs in cookbook.

### PR DESCRIPTION
**Pull request by bharat-biradar**
Fixes [#75723](https://github.com/flutter/flutter/issues/75723).

Changes proposed in this pull request:

1. Updated the takePicture() example in the cookbooks.
2. Added a comment on Xfile.

### CODE REVIEW

File: src/docs/cookbook/plugins/picture-using-camera.md [hunk 1]
```diff
@@ -179,15 +179,15 @@ FutureBuilder<void>(
 ## 5. Take a picture with the `CameraController`
 
 You can use the `CameraController` to take pictures using the
-[`takePicture()`][] method. In this example,
-create a `FloatingActionButton` that takes a picture
+[`takePicture()`][] method. The [`takePicture()`][] method returns a [`Xfile`](https://pub.dev/documentation/camera/latest/camera/XFile-class.html) which is a cross-platform simplified File abstraction. In android and IOS implementation the image taken is stored in their respective cache directories. The path of the Xfile returned points to the image stored in the cache directory of the device.
```
Comment (sfshaza2) [inline src/docs/cookbook/plugins/picture-using-camera.md:182]:
```suggestion
[`takePicture()`][] method, which returns an [`Xfile`][],
a cross-platform, simplified `File` abstraction.
On both Android and IOS, the new image is stored in their
respective cache directories,
and the `path` to that location is returned in the `Xfile`.

[`Xfile`]:  {{site.pub}}/documentation/camera/latest/camera/XFile-class.html
```

File: src/docs/cookbook/plugins/picture-using-camera.md [hunk 2]
```diff
@@ -179,15 +179,15 @@ FutureBuilder<void>(
...
 using the `CameraController` when a user taps on the button.
 
-Saving a picture requires 3 steps:
+Taking a picture requires 2 steps:
 
-  1. Ensure the camera is initialized
-  2. Construct a path that defines where the picture should be saved
-  3. Use the controller to take a picture and save the result to the path
+  1. Ensure the camera is initialized.
+  2. Use the controller to take a picture and it returns a Future<Xfile>.
```
Comment (sfshaza2) [inline src/docs/cookbook/plugins/picture-using-camera.md:190]:
```suggestion
  1. Ensure that the camera is initialized.
  2. Use the controller to take a picture and ensure that it returns a `Future<Xfile>`.
```

File: src/docs/cookbook/plugins/picture-using-camera.md [hunk 3]
```diff
@@ -204,17 +204,9 @@ FloatingActionButton(
...
-        // Find the temp directory using the `path_provider` plugin.
-        (await getTemporaryDirectory()).path,
-        '${DateTime.now()}.png',
-      );
-
-      // Attempt to take a picture and log where it's been saved.
-      await _controller.takePicture(path);
+      // Attempt to take a picture and get the file
+      // where the image was saved.
+      final image = await _controller.takePicture();
```
Comment (sfshaza2) [inline src/docs/cookbook/plugins/picture-using-camera.md:209]:
```suggestion
      // Attempt to take a picture and then get the location
      // where the image file is saved.
      final image = await _controller.takePicture();
```

**Review by sfshaza2 — APPROVED** [review]
A few edits but, otherwise, lgtm

**Comment by nt4f04uNd** [comment]
@sfshaza2 this is `XFile` and not `Xfile`, revert or patch

**Comment by nt4f04uNd** [comment]
@sfshaza i ask you to respond to my previous comment

**Comment by sfshaza2** [comment]
I did fix it, @nt4f04uNd, you just can't see it yet.  :). I hope you join [Flutter Engage](https://events.flutter.dev/)!

**Pull request merged**